build-scripts/build_rule_playbooks.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/python3
+
+import argparse
+import ssg.playbook_builder
+
+
+def parse_args():
+    p = argparse.ArgumentParser()
+    p.add_argument(
+        "--input-dir", required=True, dest="input_dir",
+        help="Input directory that contains all Ansible remediations"
+        "snippets for the product we are building. "
+        "e.g. ~/scap-security-guide/build/fedora/fixes/ansible"
+    )
+    p.add_argument(
+        "--output-dir", required=True, dest="output_dir",
+        help="Output directory to which the rule-based Ansible playbooks "
+        "will be generated. "
+        "e.g. ~/scap-security-guide/build/fedora/playbooks"
+    )
+    p.add_argument(
+        "--product-yaml", required=True, dest="product_yaml",
+        help="YAML file with information about the product we are building. "
+        "e.g.: ~/scap-security-guide/rhel7/product.yml"
+    )
+    p.add_argument(
+        "--profile",
+        help="Generate Playbooks only for given Profile ID"
```

Comment:
I would mention that we accept short profile IDs s.a. `ospp` and I would make it more explicit that this argument can only restrict the selection of profiles, i.e. that all are on by default.

Reply:
Done